android/src/main/kotlin/com/bitmovin/player/flutter/FlutterPlayerView.kt

```diff
@@ -119,6 +141,9 @@ class FlutterPlayerView(
         sink = null
     }
 
+    private val isPictureInPictureSupported: Boolean
+        get() = Build.VERSION.SDK_INT >= Build.VERSION_CODES.O
+
```

Comment:
NIT: could be a top level function in the file to make it clear that it does not need any state from the view.

Reply:
https://github.com/bitmovin/bitmovin-player-flutter/pull/87/commits/4f1efd2a592845b1c63e451e62a6905b4e2a20b5